### ISSUE TITLE
Correct issues spotted by translators

### ISSUE THREAD

**Issue by Minoru** [newsboat/newsboat#1354]
This issue is a collection of problems in our translatable strings:

- https://github.com/newsboat/newsboat/pull/1341#discussion_r537651337 (not clear that "between" is an operator and shouldn't be translated)
- https://github.com/newsboat/newsboat/pull/1349#discussion_r538748819 (one of the help messages is not a single sentence like all others)
- https://github.com/newsboat/newsboat/pull/1352#discussion_r540093840 (not clear if "quoted string" in "one of: quoted string, interval, number" should be translated)
- https://github.com/newsboat/newsboat/issues/1354#issuecomment-742586235 (some messages list details after a colon, while others do it in parenthesis)
- https://github.com/newsboat/newsboat/issues/1354#issuecomment-742586235 (after the "Error: " prefix, some messages start the explanation with uppercase, while others use lowercase)

These should be fixed after 2.22 is out, and before 2.23 translation period starts.

(@der-lyse, you're reviewing plenty of translations too — please add to this issue if you spot things that, ideally, should be fixed in the POT file, not in each and every PO file)

**Comment by der-lyse** [comment]
The only other minor thing I noticed, is a kind of inconsistency where newer error reasons are put in parenthesis rather than appending them after a colon, e.g.

```
Error: Failed to read URLs from file "%s" (%s)
vs.
Error: applying the filter failed: %s
```

Sometimes filenames are quoted, sometimes not:

```
Error: couldn't write article to file %s
vs.
Error: Failed to parse OPML file "%s"
```

Some use single quotes instead:

```
XDG: configuration directory '%s' not accessible, using '%s' instead.
```

Others follow the backtick and single quote approach:

```
Fatal error: couldn't create configuration directory `%s': (%i) %s
```

I'm a fan of quoting things, whenever possible and there's no better way to make them stand out, e.g. by changing the font or style. Since we're in the terminal world here, quotes are the way go in my opinion. A while back I started to introduce the typographically correct quotes `„` and `“` as defined by German type setting standards for new messages in _de.po_. But there are still several regular double ASCII quotes `"` left (maybe other variants, too). Maybe in the upcoming winter holidays I'll find some time to tackle this (no promises though ;-)).

This small assortment of examples also shows that we don't follow a uniform rule about whether to continue with upper or lower case after the colon for further information.
